here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/models/storage/ContextWriteFeatures.java

```diff
@@ -38,7 +37,7 @@ public abstract class ContextWriteFeatures extends WriteRequest {
    * The list of features passed as context, as part of Write request
    */
   @ApiStatus.AvailableSince(NakshaVersion.v2_0_11)
-  private @Nullable List<@NotNull NakshaContext> context;
+  private @Nullable List<@NotNull NakshaFeature> context;
```

Comment:
So `NakshaContext` won't exist anymore? What's the thought behind that?

Reply:
Putting `NakshaContext` there was my previous mistake when working on lib-core. The javadoc for this class says "Abstract class representing WriteFeatures request along with **list of features as context**"